pyqtgraph/graphicsItems/PlotCurveItem.py

```diff
@@ -99,14 +99,14 @@ def setCompositionMode(self, mode):
 
         ============================================  ============================================================
         **Most common arguments:**
-        QtGui.QPainter.CompositionMode_SourceOver     Default; image replaces the background if it
+        QtGui.QPainter.CompositionMode.CompositionMode_SourceOver     Default; image replaces the background if it
                                                       is opaque. Otherwise, it uses the alpha channel to blend
                                                       the image with the background.
-        QtGui.QPainter.CompositionMode_Overlay        The image color is mixed with the background color to
+        QtGui.QPainter.CompositionMode.CompositionMode_Overlay        The image color is mixed with the background color to
                                                       reflect the lightness or darkness of the background.
-        QtGui.QPainter.CompositionMode_Plus           Both the alpha and color of the image and background pixels
+        QtGui.QPainter.CompositionMode.CompositionMode_Plus           Both the alpha and color of the image and background pixels
                                                       are added together.
-        QtGui.QPainter.CompositionMode_Multiply       The output is the image color multiplied by the background.
+        QtGui.QPainter.CompositionMode.CompositionMode_Multiply       The output is the image color multiplied by the background.
```

Comment:
These seem to break the docstring alignment.

Reply:
ooooooooof... yea I'm going to have to take a closer look at those :|